great_expectations/util.py

```diff
@@ -1782,6 +1784,7 @@ def get_context(  # noqa: PLR0913
     cloud_access_token: str | None = ...,
     cloud_organization_id: str | None = ...,
     cloud_mode: bool | None = ...,
+    mode: str | None = None,
```

Comment:
```suggestion
    mode: None = ...,
```

Reply:
I don't know this convention. Cargo culting for now.